app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java

```diff
@@ -315,6 +326,23 @@ public boolean onMarkerClick(Marker marker) {
         return true;
     }
 
+    private void setupPOIListListener() {
+        mViewModel.getListOfPOI().observe(getViewLifecycleOwner(), new Observer<List<WalkingPoint>>() {
+            @Override
+            public void onChanged(List<WalkingPoint> walkingPoints) {
+                for(WalkingPoint point: walkingPoints){
+                    LatLng latLng = new LatLng(point.getCoordinate().getLongitude(),point.getCoordinate().getLatitude());
+                    MarkerOptions markerOptions = new MarkerOptions()
+                            .position(latLng)
+                            //TODO: Change the icon to be the same as the POI icon
```

Comment:
Do you still need this TODO ?

Reply:
I do. These todos will be cleared before this task is closed. I am keeping them here as checkpoints of what's left to do.

Reply:
can you add the issue in the zenhub and add it to the comment please?

Reply:
It's really part of this task tho. Like a subtask. This normally would not be pushed to master but for the sake of smaller more comprehensible PRs I thought of adding it. Do you still think a new issue needs to be created?

Reply:
ok sounds good so I will approve, as long as you push the rest of the code later